apps/web/src/pages/SearchRequests/SearchPage/utils.ts

```diff
@@ -23,13 +23,13 @@ import {
   formatClassificationString,
   poolMatchesClassification,
 } from "~/utils/poolUtils";
-import nonExecutiveITClassifications from "~/constants/nonExecutiveITClassifications";
+import classificationsAvailable from "~/constants/classificationsAvailableForSearch";
 import { positionDurationToEmploymentDuration } from "~/utils/searchRequestUtils";
 
 export const getAvailableClassifications = (
   pools: Pool[],
```

Comment:
I forgot that we had this business logic, where we only show classifications which have at least one published pool. That doesn't quite match the AC... I'll check Gray about whether we want to keep this logic or ditch it entirely.